Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -107,6 +116,19 @@ public async Task RemoveResponse(DiscordRequest request, Contexts contexts)
         [DiscordCommand("responses")]
         public async Task Responses(DiscordRequest request, Contexts contexts)
         {
+            // do testu obsługi reakcji:
```

Comment:
lepszy do oceny byłby bardziej praktyczny scenariusz

nam nie chodzi o to czy informacje o reakcjach są dostępne -> bo wiemy że są dostępne od początku i wiemy że ten mechanizm ma prawo działać

większym problemem jest sama wygoda użytkowania i niezawodność mechanizmu

Reply:
lepszym przykładem będzie dodawanie roli, jeśli ktoś da określoną reakcje
wybierz jakąś role z dostępnych na serwerze, powiedzmy że role "nowy" jak ktoś doda reakcje przykładowo `:thumbsup:`

Reply:
i wtedy zobaczymy czy mechanizm jest prosty, przyjemny i intuicyjny w obsłudze 

czy trzeba sie z nim męczyć -> pamiętać o szczegółach